crates/sui-types/src/object.rs

```diff
@@ -95,7 +95,7 @@ impl MoveObject {
     pub fn has_public_transfer(&self) -> bool {
         self.has_public_transfer
     }
-
+    // TODO: should this panic?
```

Comment:
how is the contents initialized? let's have a Move person checking that.

Reply:
This length is indeed not enforced, and a malicious `MoveObject` can have a contents field deserialized as `Bytes` that would be too small and would panic the node on computing this. I would open an issue for this. /cc @sblackshear @tnowacki